src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -175,6 +175,40 @@
                     ]
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/raft/{namespace}/{topic}/{partition}",
+            "operations": [
+                {
+                    "method": "GET",
+                    "summary": "Get raft states for leader and followers",
```

Comment:
Can we add an information that this must be executed against the leader or it will be redirected.